### ISSUE TITLE
Web Manifest not loaded when using HTTP basic auth

### ISSUE THREAD

**Issue by eggwhalefrog** [thelounge/thelounge#4227]
- _Node version: 12.18.3_
- _Browser version: Chromium 90.0.4430.93_
- _Device, operating system: Desktop, manjaro_
- _The Lounge version: 4.2.0_

---
The file 'thelounge.webmanifest' fails to load with a 401 Unauthorized error when hosted behind HTTP basic auth.
After adding 'crossorigin="use-credentials"' as described here https://github.com/aws-amplify/amplify-console/issues/195 the manifest is loaded as expected and it's possible to install thelounge as a PWA.

I'm running thelounge in a docker container and this worked for me as a temporary fix: `sed -i 's/link rel="manifest"/link rel="manifest" crossorigin="use-credentials"/g' /usr/local/share/.config/yarn/global/node_modules/thelounge/client/index.html.tpl`

**Comment by dgw** [comment]
Oh, yep. I had to fix this same issue a few months ago in afaundez/jekyll-favicon#31.

**Comment by 9p4** [comment]
See #4447:

TL;DR: you need to whitelist some paths with HTTP basic auth.
